codemon/codemon.py

```diff
@@ -19,11 +19,8 @@ def main():
       if arg == "init":
         if sys.argv[countArg] == '-n':
           file = sys.argv[countArg+1]
-          path = '.'
-          f = open(path + '/' + file + '.cpp',"w+")
           template = template_cpp()
-          f.write(template)
-          f.close()
+          write_to_file(f'{file}.cpp', template)
           print(colored.yellow("Created "+file+'.cpp'))
```

Comment:
Can you create another function `initSingleFile(fileName, template)` to refactor this part ? It makes the code a lot cleaner here.